internal/xds/translator/testdata/in/ratelimit-config/header-and-cidr-matches.yaml

```diff
@@ -0,0 +1,36 @@
+name: "first-listener"
+address: "0.0.0.0"
+port: 10080
+hostnames:
+- "*"
+path:
+  mergeSlashes: true
+  escapedSlashesAction: UnescapeAndRedirect
+routes:
+- name: "first-route"
+  traffic:
+    rateLimit:
+      global:
+        rules:
+        - headerMatches:
+          - name: "x-user-id"
+            exact: "one"
+          - name: "x-user-id"
+            exact: "two"
```

Comment:
Nit: This is not a good example as "x-user-id=one && x-user-id=two" will always be false.

This raises a broader question: should we allow the same header appear multiple times in the `Headers` match conditions? @arkodg

Reply:
the same header multiple times is valid, for e.g. https://github.com/envoyproxy/gateway/pull/4452

Reply:
This is ver difficult to understand:
```
        - clientSelectors:
            - headers:
                - name: x-user-name
                  type: Distinct
                - name: x-user-name
                  type: Exact
                  value: admin
                  invert: true
          limit:
            requests: 3
            unit: Hour                  
 ```
 
I prefer:
 
 ```
         - clientSelectors:
            - headers:
                - name: x-user-name
                  type: Exact
                  value: admin
                  invert: true
           limit:
            requests: 3
            unit: Hour                 
 ```

Reply:
updated the test, add a two different header names case